lib/metadata/src/metadata.cpp

```diff
@@ -0,0 +1,173 @@
+#include "irods/metadata.hpp"
+
+#include "irods/rcConnect.h"
```

Comment:
Remove this include statement.

This isn't needed because we don't access anything inside of the `rxComm_t`.